src/registry/validations.py

```diff
@@ -0,0 +1,119 @@
+import json
+import re
+from datetime import date
+from pathlib import Path
+
+
+def validate_commit(commit):
+    # Check the hash commit length
+    hash_size = 40
+    if len(commit) != hash_size:
+        return """Invalid commit: 
+            The commit hash should be a 40-character string."""
```

Comment:
Is this message useful to the API user? after all the user does not control the commit hash. Perhaps it should teach the user how to properly capture the commit hash string.

Reply:
Done!